paper/src/main/java/rocks/gravili/notquests/paper/conversation/ConversationManager.java

```diff
@@ -55,6 +52,9 @@
 import rocks.gravili.notquests.paper.structs.variables.VariableDataType;
 
 public class ConversationManager {
+
+  public static final Map<Integer, List<UUID>> CONVERSATIONS_IN_PROGRESS = new HashMap<>();
```

Comment:
hmm what this does is not quite clear from the naming (that it stores npc IDs). Also, is it necessary, considering all open conversations are already stored in "openConversations"?

Also make sure not to use static here (or pretty much anywhere in notquests)

Reply:
Just to easily retrieve all the conversations that involve this NPC: the conversations can have several speakers and it is much more complex to retrieve the one that involves the current NPC.